pyproject.toml

```diff
@@ -101,6 +101,7 @@ lint.ignore = [
     "E501",   # line too long
     "F541",   # f-string-missing-placeholders
     "RUF012", # mutable-class-default
+    "A005",   # Module shadows a Python standard-library module (needed for signal)
```

Comment:
You could move this to per-file-ignores if it's just one file

Reply:
good point. fixed.